docker-compose-generator/docker-fragments/opt-add-joinmarket.yml

```diff
@@ -3,10 +3,12 @@ version: "3"
 services:
   joinmarket:
     container_name: joinmarket
-    image: btcpayserver/joinmarket:0.9.4
+    image: btcpayserver/joinmarket:0.9.5
```

Comment:
Would be nice to pin this to the image hash as well. Just an additional security measure to avoid overwritten images since afaik they aren't immutable in docker hub